akka-epl/src/main/scala/com/epl/akka_v1/HttpClient.scala

```diff
@@ -0,0 +1,85 @@
+package com.epl.akka_v1
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.marshalling.Marshal
+import akka.http.scaladsl.model.headers.BasicHttpCredentials
+import akka.http.scaladsl.model.{HttpMethods, HttpRequest, RequestEntity, headers}
+import akka.http.scaladsl.unmarshalling.Unmarshal
+import akka.stream.ActorMaterializer
+import akka.util.ByteString
+import com.epl.akka.DocumentType
+
+import scala.concurrent.Future
+
+object HttpClient {
+
+  implicit val system = ActorSystem()
+  implicit val materializer = ActorMaterializer()
+  implicit val ec = system.dispatcher
+
+  final val config = system.settings.config
+
+  private val authorization = headers.Authorization(BasicHttpCredentials(config.getString("cloudant.username"), config.getString("cloudant.password")))
+
+
+  /**
+    * POST call to Cloudant database API
+    * @param jsonString
+    * @return
+    */
+  def post(jsonString: String): Future[String] = {
+    val responseEntity = for {
+      request <- Marshal(jsonString).to[RequestEntity]
+      response <- Http().singleRequest(HttpRequest(
+        method = HttpMethods.POST,
+        uri = config.getString("cloudant.post_url"),
+        headers = List(authorization),
+        entity = request))
+      entity <- Unmarshal(response.entity).to[ByteString]
+    } yield entity
+
+    responseEntity.mapTo[String]
```

Comment:
This doesn't look right to me, you cannot just typecast a ByteString to a String, you should either Unmarshal a string above if possible, or you would have to `.map(byteString => byteString.utf8String)` to parse the bytes into a `String`